go/go.mod

```diff
@@ -59,6 +59,7 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51 // indirect
 	github.com/kr/text v0.2.0 // indirect
+	github.com/linkedin/goavro/v2 v2.12.0 // indirect
```

Comment:
Might I suggest `github.com/hamba/avro/v2/` as it seems to be about an order of magnitude faster than goavro....?

Reply:
had a question about licence compatibility but a Google search answered it, will look to moving avro reader to Hamba